### ISSUE TITLE
Feature request: `now whoami`

### ISSUE THREAD

**Issue by renatorib** [vercel/vercel#585]
Simple but effective command to know which account you are in.

```sh
$ now whoami
> renatorib
```

(I don't know how Teams works, but perhaps it can help with Teams too)

**Comment by rauchg** [comment]
Yep. `now switch` kinda tells you already, but not sure if it does if you have no teams

**Comment by chabou** [comment]
Seems fixed by https://github.com/zeit/now-cli/commit/24ce1c2b983e0372a3d0b2f7ab5d2acdb716c5d8

However, I need the same thing for current team. I saw a revert and a new version without team (for consistency).

My problem is: I want to show (in prompt) current team or current user if there is no team. I had to parse `.now.json` with 3 different ways (js, ruby, python) to adapt to user environnement. It will be really better to exec `now whoami` or/and `now teams current` and not parsing `now` internals.

This is my current solution: https://github.com/chabou/pure-now/blob/v1.5.4/now_functions.zsh
And there is a PR to `hyper-statusline` by @maxbeatty with the same need: https://github.com/henrikdahl/hyper-statusline/pull/49/commits/30f9f4482b2ccf919c14025f3233625465a928af

I can make a PR for this new command if you give me its name:
* `now teams current`
* `now teams info`
* `now teams show`
* `now teams status`
